benches/common.rs

```diff
@@ -0,0 +1,98 @@
+//! Common benchmark helpers shared by multiple benchmarks.
```

Comment:
To avoid confusing this with an actual benchmark, could this be placed at `benches/common/mod.rs`?